CONTRIBUTING.md

```diff
@@ -84,6 +84,22 @@ To contribute a patch, the workflow is as follows:
   1. Create topic branch
   1. Commit patches
 
+For GUI related issues or pull requests, the https://github.com/bitcoin-core/gui repository should be used.
+For all other issues and pull requests, the https://github.com/bitcoin/bitcoin main repository should be used.
+
+As a rule of thumb, everything that only modifies `src/qt` is a gui-only pull request. However,
+* Global refactoring or other global changes that touch GUI code, as well as
+  non-GUI code, the main repository should be used.
```

Comment:
- "other global" could be "other transversal"
- can omit "that touch...as non-GUI code,"
- s/the ... be used./should be opened in bitcoin/bitcoin./